maxim/maxim.py

```
@@ -48,6 +48,18 @@
 
 
 class ConfigDict(TypedDict, total=False):
+    """
+    A class representing the configuration for the Maxim SDK.
+
+    Attributes:
+        api_key (Optional[str], optional): The API key for the Maxim instance. Defaults to None.
+        base_url (Optional[str], optional): The base URL for the Maxim instance. Defaults to None.
+        cache (Optional[MaximCache], optional): The cache to use for the Maxim instance. Defaults to None.
```

Pylint (3.3.7)

[convention] Line too long (106/100)

(C0301)